### DISCUSSION TITLE
Dump reports from shell

### DISCUSSION THREAD

**Discussion by khaledhasanzami** [The-OpenROAD-Project/OpenROAD-flow-scripts#845] [Q&A]
I wanted to to export certain reports (timing, congestion) from shell. But I cant save it into a file. Can you suggest any?

**Comment by maliberty** [comment]
Can you use the log files from OR for your purpose?  We also generate a metrics file if that is what you need.

**Comment by maliberty** [comment]
What is your goal for which you need this?  Having context can help with generating solutions.

**Reply by khaledhasanzami** [reply]
I wanted to dump congestion report manually from shell.
